rpc/sosapi_client.c

```diff
@@ -244,6 +244,7 @@ struct dsos_session_s {
 		ERROR
 	} state;
 	pthread_mutex_t lock;	/* Mutex across entire session */
+	pthread_cond_t  cond;	/* For busy/idle condition */
```

Comment:
Where is this initialized?

Reply:
Oops. My bad. I just updated the patch. It is initialized here: https://github.com/ovis-hpc/sos/pull/92/files#diff-987337fa192d86403df996923aac73afaf89f615abb01dcea6e09dddeb099a40R1229